'References' tab shows error (nullPointerException) when selecting more than one row

### Discussed in https://github.com/orgs/dbeaver/discussions/18534

<div type='discussions-op-text'>

<sup>Originally posted by **Amitkad** December 19, 2022</sup>
Few days ago dbeaver asked to make an update - I approved it (and I think it also installed\updated a driver)
I'm not 100% that the issue started than, but I am 90% sure about it.

The issue:

* When I am selecting more than one row, the 'References'  panel show a "NullPointerException" (full details below).
* the References panel works fine when selecting single row.
* It happens on all my DBs and connections

![image](https://user-images.githubusercontent.com/7620041/208439611-b4046aea-7699-4772-87f2-30d85b31089e.png)

**Exception**:
org.jkiss.dbeaver.DBException
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:232)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5035)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.NullPointerException
	at org.jkiss.dbeaver.model.impl.sql.AbstractSQLDialect.isQuotedIdentifier(AbstractSQLDialect.java:400)
	at org.jkiss.dbeaver.model.impl.sql.AbstractSQLDialect.getQuotedIdentifier(AbstractSQLDialect.java:410)
	at org.jkiss.dbeaver.model.DBUtils.getQuotedIdentifier(DBUtils.java:159)
	at org.jkiss.dbeaver.model.DBUtils.getQuotedIdentifier(DBUtils.java:154)
	at org.jkiss.dbeaver.model.impl.sql.StandardSQLDialectQueryGenerator.getConstraintCondition(StandardSQLDialectQueryGenerator.java:344)
	at org.jkiss.dbeaver.model.impl.sql.StandardSQLDialectQueryGenerator.appendConditionString(StandardSQLDialectQueryGenerator.java:171)
	at org.jkiss.dbeaver.model.impl.sql.StandardSQLDialectQueryGenerator.appendConditionString(StandardSQLDialectQueryGenerator.java:107)
	at org.jkiss.dbeaver.model.impl.sql.StandardSQLDialectQueryGenerator.appendConditionString(StandardSQLDialectQueryGenerator.java:92)
	at org.jkiss.dbeaver.model.impl.sql.StandardSQLDialectQueryGenerator.appendQueryConditions(StandardSQLDialectQueryGenerator.java:63)
	at org.jkiss.dbeaver.model.sql.SQLUtils.appendQueryConditions(SQLUtils.java:1052)
	at org.jkiss.dbeaver.model.impl.jdbc.struct.JDBCTable.readData(JDBCTable.java:162)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:173)
	... 4 more


I tried to:
* look for the issue and solution online
* reset driver to default
* reoper dbeaver 
* and more...

if anyone have an idea i'll be glad!</div>

DBeaver 22.3.0

--- a/plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/StandardSQLDialectQueryGenerator.java
+++ b/plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/StandardSQLDialectQueryGenerator.java
@@ -340,9 +340,14 @@ public class StandardSQLDialectQueryGenerator implements SQLDialectQueryGenerato
                     Object[] array = ((Object[]) value);
                     for (int i = 0; i < array.length; i++) {
                         if (i > 0) {
-                            conString.append(" OR");
-                            conString.append(' ').append(DBUtils.getQuotedIdentifier(dataSource,
-                                constraint.getAttributeLabel())).append(' ');
+                            conString.append(" OR ");
+                            conString.append(DBUtils.getQuotedIdentifier(
+                                dataSource,
+                                CommonUtils.isEmpty(constraint.getAttributeLabel()) ?
+                                    constraint.getAttributeName() :
+                                    constraint.getAttributeLabel()
+                            ));
+                            conString.append(' ');
                         }
                         conString.append(operator.getExpression());
                         String strValue = getStringValue(dataSource, constraint, inlineCriteria, array[i]);

